src/app/features/home/onboarding/tutorial/tutorial.page.scss

```
@@ -24,6 +24,24 @@ ion-content {
   }
 }
 
+.onboarding-container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
+  background-color: #0000d8;
+}
+
+.onboarding-logo {
+  width: 95%;
+  margin: auto 20;
+  margin-bottom: 20px;
+}
+
+.onboarding-text {
+  flex: 1; // 高度自動伸展至所有剩餘的空間
```

```suggestion
  flex: 1; // Fill remaining space of the flex-direction (column)
```
We would like the repository's doc, descriptions and comments to be English-only